src/test/suite/snippets/stageAutocompleter.test.ts

```diff
@@ -41,8 +41,6 @@ const STAGE_LABELS = [
 ];
 
 suite('Stage Autocompleter Test Suite', () => {
-  vscode.window.showInformationMessage('Starting tests...');
```

Comment:
Hehe I was about to ask you if we really want to have these messages but you already deleted them 👍